Packs/CommonPlaybooks/Playbooks/playbook-Handle_False_Positive_Alerts_README.md

```diff
@@ -0,0 +1,38 @@
+This playbook handles false positive alerts.
```

Comment:
This playbook handles false positive alerts.
The playbook will create an alert exclusion, exception or add a file to allowlist based on the alert fields and playbook inputs.